LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -137,7 +137,15 @@ function addHeader(column) {
                     $('#'+data.cellContainerID).html(blob[data.recordID].recordResolutionData.lastStatus);
                 }
             }});
-        break;
+            break;
+        case 'resolvedBy':
+                        leafSearch.getLeafFormQuery().join('recordResolutionBy');
```

Comment:
Need the indentation fixed here.  Looks like there may be some invisible characters stuck in there that your local machine isn't seeing but are getting passed up.  Normally I wouldn't tag this so late in the Sprint, but since you're going to be in here anyway.

Reply:
indention fixed